### PR TITLE
chore: rename reaction to moc

### PR DESCRIPTION
**Pull request by Akarshit**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
 # Leave this here unless you understand the implications.
 # See this issue for details:
-# https://github.com/reactioncommerce/reaction/pull/5118
+# https://github.com/mailchimp-open-commerce/api-server/pull/5118
 !.git/modules/imports/plugins/custom
 
 .build.log
 Dockerfile
-.reaction/docker/reaction.*
+.moc/docker/reaction.*
```
Comment (jrw421) [inline .dockerignore:14]: Should we change the reaction in docker as well?

File: .github/ISSUE_TEMPLATE/questions.md [hunk 2]
```diff
@@ -10,5 +10,5 @@ labels: 'questions'
 
 Please contact us instead. We are active in gitter and twitter:
 
-- [Gitter](https://gitter.im/reactioncommerce/reaction)
+- [Gitter](https://gitter.im/mailchimp-open-commerce/api-server)
```
Comment (jrw421) [inline .github/ISSUE_TEMPLATE/questions.md:13]: Are we changing our Gitter url?

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+This code of conduct applies to all spaces managed by Mailchimp Open Commerce. This includes our [development chat room](https://gitter.im/mailchimp-open-commerce/api-server), [forums](https://forums.reactioncommerce.com), [blog](https://blog.reactioncommerce.com), mailing lists, [issue tracker](https://github.com/mailchimp-open-commerce/api-server/issues), [project boards](https://github.com/mailchimp-open-commerce/api-server/projects), Mailchimp Open Commerce events and meetups, and any other forums or service created by the core project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
```
Comment (jrw421) [inline CODE_OF_CONDUCT.md:11]: Same Gitter question as above ^

File: README.md [hunk 4]
```diff
@@ -90,35 +90,35 @@ Use an external GraphQL client to test http://localhost:3000/graphql. GraphQL Pl
 
 ## Get help & contact the team
 
-- [Gitter chat](https://gitter.im/reactioncommerce/reaction)
-- Report security vulnerabilities to <mailto:security@reactioncommerce.com>: [Security reporting instructions](https://docs.reactioncommerce.com/reaction-docs/trunk/reporting-vulnerabilities)
-- Request features in this [repository](https://github.com/reactioncommerce/reaction-feature-requests/)
+- [Gitter chat](https://gitter.im/mailchimp-open-commerce/api-server)
```
Comment (jrw421) [inline README.md:93]: Same Gitter question as above ^

**Review by jrw421 — CHANGES_REQUESTED** [review]
I didn't see any updates to the ChangeLog - we should add a section explaining the naming changes with the new version :)

File: .github/ISSUE_TEMPLATE/questions.md (hunk 2 above)
Comment (Akarshit) [reply .github/ISSUE_TEMPLATE/questions.md:13]: I was hoping that they would update once we rename the org/repo. I can try to get a confirmation on this one.

**Pull request closed without merging**